sagemaker_neo_compilation_jobs/tensorflow_distributed_mnist/tensorflow_distributed_mnist_neo.ipynb

```diff
@@ -123,11 +119,12 @@
     "\n",
     "mnist_estimator = TensorFlow(entry_point='mnist.py',\n",
     "                             role=role,\n",
-    "                             framework_version='1.11.0',\n",
+    "                             framework_version='1.15.3',\n",
```

Comment:
Why are we using TF v<2?

Reply:
Neo currently doesn't support TF 2.x version. So, we're using 1.15 which is the current TF 1.x version.